dist/index.d.ts

```diff
@@ -4781,7 +4781,7 @@ interface Spawning {
      * An array with the spawn directions
      * @see http://docs.screeps.com/api/#StructureSpawn.Spawning.setDirections
      */
-    directions: DirectionConstant[];
+    directions: DirectionConstant[] | undefined;
```

Comment:
```suggestion
    directions?: DirectionConstant[];
```
To avoid re-building